modules/n1ql/pages/n1ql-language-reference/indexing-arrays.adoc

```diff
@@ -288,7 +288,7 @@ WHERE ANY v IN schedule SATISFIES v.flight LIKE 'UA%' END;
 
 .Partial index (with WHERE clause) of individual attributes from selected elements (using WHEN clause) of an array
 ====
-C{counter:example-counter}: Create an index on flights from San Francisco scheduled in the first 4 days of the week
+.C{counter:example-counter}: Create an index on flights from San Francisco scheduled in the first 4 days of the week
 [[C2,C{example-counter}]]
 [source,N1QL]
 ----
```

Comment:
Antora 3.0.0 alpha 2. The example counter works in the running text, but is not being resolved in xref labels.

![Screen Shot 2021-05-27 at 00 18 24](https://user-images.githubusercontent.com/39966290/119745061-47d4fa80-be85-11eb-8b98-9670df5ee811.png)

Reply:
For comparison, the `{example-number}` attribute _is_ resolved in xrefs when using Antora 3.0.0 alpha 2.

![Screen Shot 2021-05-27 at 00 56 38](https://user-images.githubusercontent.com/39966290/119745644-756e7380-be86-11eb-8042-793187a4d293.png)

Reply:
Yes, this fix for Asciidoctor 2 doesn't seem to also work for Asciidoctor 1.5.x. Since the plan is to upgrade immediately, I think we can just make the leap. If we need to accommodate both Asciidoctor versions, then it would be necessary to add ifeval logic around the two alternatives.

Reply:
Okay, I updated to Antora 3.0.0 alpha 5, and the example counter is now resolved in xref labels, but unfortunately it's still not working quite right. Looking at the same piece of text:

![Screen Shot 2021-05-27 at 11 54 41](https://user-images.githubusercontent.com/39966290/119814655-91076780-bee2-11eb-944e-c50a2ce2d2bd.png)

The first *C1* (highlighted) should be *C2*, as in [the comment above](https://github.com/couchbase/docs-server/pull/1975#discussion_r640195998).